cli/dcos-spark/vendor/github.com/mesosphere/dcos-commons/cli/README.md

```diff
@@ -36,12 +36,12 @@ cd sdk/cli/
 To run the compiled CLI (the example below uses Linux, pick the correct binary for your platform):
 
 ```bash
-./dcos-service-cli-linux hello-world -h
+./dcos-service-cli-linux helloworld -h
```

Comment:
is it coming from the updated dependencies? we should think about including those as a git module to avoid this kind of diffs in PRs but this is out of the scope of this PR.